src/Core/Managed/PCL/Core.PCL.csproj

```diff
@@ -25,20 +24,65 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
   <ItemGroup>
-    <None Include="packages.config" />
+    <None Include="project.json" />
```

Comment:
It uses .csproj to build and project.json to specify the references (NuGet packages)